src/main/java/com/hazelcast/aws/AwsProperties.java

```diff
@@ -82,6 +81,11 @@
      */
     CONNECTION_TIMEOUT_SECONDS("connection-timeout-seconds", INTEGER, true),
 
+    /**
+     * Number of retries while connecting to AWS Services. Its default value is 10.
```

Comment:
is this Metadata Service? giving more info here could be helpful for other people.
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html

Reply:
Good point. Actually, it's about all AWS services we use (DescribeInstances and InstanceMetadata). Added a comment.